app/integration/runtime/src/main/java/io/syndesis/integration/runtime/handlers/SplitAggregateStepHandler.java

```diff
@@ -82,10 +82,10 @@ public boolean canHandle(Step step) {
         return Optional.of(route);
     }
 
-    public static class EndHandler implements IntegrationStepHandler {
+    public static class AggregateStepHandler implements IntegrationStepHandler {
```

Comment:
I know that is quite a dummy class and it make sense now but I would move the aggregate step (and related utilities) to a separate class so we can have a generic and reusable step to be used for next iteration

Reply:
Agreed! Pushed it to this PR